obspy/signal/tests/test_konnoohmachi.py

```diff
@@ -134,19 +134,25 @@ def test_konno_ohmachi_smoothing(self):
                                              max_memory_usage=0)
         # XXX: Why are the numerical inaccuracies quite large?
         np.testing.assert_almost_equal(smoothed_1, smoothed_2, 3)
-        # Test the non-matrix mode for single spectra.
+        # Test using a pre-computed smoothing matrix
+        with np.errstate(all='ignore'):
```

Comment:
I personally don't like this too much but it is a consequence of my own old code so I cannot complain too much ;-) Maybe it would be possible to explicitly avoid taking the logarithm of zero and also the division by zero in the `konno_ohmachi_smoothing_window()` function so if completes without raising numpy warnings?